bindings/python/upgrade.md

```diff
@@ -4,8 +4,7 @@
 
 Because of [a TLS lib issue](https://github.com/apache/opendal/issues/3650), we temporarily disable the `services-ftp` feature.
 
-
-# Upgrade to v0.42
+## Upgrade to v0.42
```

Comment:
This should be `# Upgrade to v0.42`

Reply:
in md standard only one # h1 in one file?

Reply:
I prefer keep existing format.

Reply:
copy